docs/roles/RoleNORManager.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: doc
+permalink: /role/nor-manager
+title: OBO Role: New Ontology Request (NOR) Manager
+---
+
+The New Ontology Request (NOR) Manager has these basic duties:
+
+1. Receiving `new ontology` requests on OBO and acknowledging the receipt
+2. Including the ontology into the OBO New Ontology Request (NOR) Dashboard
+3. Supporting the New Ontology requestor to passing the OBO Dashboard
+4. Assigning a provisional reviewer from the OBO Operations Committee
+5. Removing the accepted ontology from the Dashboard upon acceptance
+
+
+### Receiving `new ontology` requests on OBO and acknowledging the receipt
+
+- The NOR Manager MUST actively listen to issues on the [OBO Issue Tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues).
+- When a new ontology request is received, and the wrong issue template was used, the New Ontology Request (NOR) Manager must inform the NOR requestor to use the correct issue template and close the wrongly submitted issue.
+- When a valid NOR request is received the NOR Manager acknowledges the receipt with the following message:
+    ```markdown
+    Dear @GITHUBNAME,
+    
+    Thank you for your submission. The review will be executed as a two stage process. 
+    First, you will have to pass [OBO NOR Dashboard](https://obofoundry.org/obo-nor.github.io/dashboard/index.html). Pass means that no check _apart from `Users` may be red_.
+    After you have successfully passed the Dashboard you will be assigned an OBO Operations committee member to review the ontology.
+    
+    Usually, the review will result in an opportunity to you to improve the ontology.
+    When the reviewer believes the ontology is ready for presentation to the OBO Operations Committee, they will present
+    your ontology during an OBO Operations Call. This gives other members of the
+    committee the opportunity to assess your work.
+    
+    When a decision is reached by the committee you will be informed here on the issue tracker.
+    The process can take any number of weeks or months, depending on the case at hand.
+    Please inform us about any reasons you might have for increased urgency.
+    
+    You will be informed once your ontology is loaded in the OBO NOR Dashboard.
+    
+    Good luck.
+    ```
+
+### Including the ontology into the OBO New Ontology Request (NOR) Dashboard
+
+
+    After the dashboard is run, the OBO dashboard administrator informs the submitter about the need to fix the issues revealed by the dashboard, noting this is not part of the review itself, just a precursor, and that upon completion, a liaison will be assigned.
+    At the next OBO Foundry Operations Committee conference call (hereafter, "Operations call"), a liaison is selected to be responsible for the issue. This liaison becomes familiar with the new ontology and rallies the appropriate people to provide feedback.
```

Comment:
this seems to be straying from the NOR Manager role and going deep into the ontology review & acceptance process. consider moving the text here and to the end of this document to a more appropriate document.